src/contracts/core/AllocationManager.sol

```diff
@@ -59,7 +66,9 @@ contract AllocationManager is
     function slashOperator(
         address avs,
         SlashingParams calldata params
-    ) external onlyWhenNotPaused(PAUSED_OPERATOR_SLASHING) checkCanCall(avs) {
+    ) external onlyWhenNotPaused(PAUSED_OPERATOR_SLASHING) {
```

Comment:
How explicit do we want to be with the `validateAVS` modifiers to check the avs input _is_ an AVS?
For example registerForOperatorSets has the isAVS check but slashOperator doesn't. For registering for an operatorSet, it has to exist implying it has to have been created, which also implies that the AVS is registered (since you must be a registered AVS to create operatorSets). Similarly, the `slashOperator` function also doesn't have the isAVS check but for there to be any slashable allocations in the first place, the AVS had to have been registered too.

Reply:
I figured we don't need it for `registerForOperatorSets` (this was erroneously checked before), `slashOperator` and `modifyAllocations` since we validate the existence of the operatorSet, which implies the avs must already have registered.

Reply:
So basically, if we validate the opSet exists, we don't need to validate that the avs IS an avs